modules/ROOT/pages/changelog.adoc

```diff
@@ -5,6 +5,15 @@
 NOTE: This is the {productname} Community version changelog. For information about the latest {cloudname} or {enterpriseversion} Release, see: xref:release-notes.adoc[{productname} Release Notes].
 
 
+== 6.7.2 - 2023-10-25
+
+=== Fixed
+* Applying heading formatting to the content of the `summary` element extended its application to the content of the parent `details` element.
+* The function `getModifierState` did not work on events passed through the editor as expected.
+* List items containing a nested list between two text caused some list operations to fail.
```

Comment:
```suggestion
* Indenting or outdenting a list item that contained non list item siblings after it would result in those siblings being removed.
```